DNN Platform/Library/Entities/Modules/ModuleController.cs

```diff
@@ -1902,6 +1902,21 @@ public void MoveModule(int moduleId, int fromTabId, int toTabId, string toPaneNa
             //Move the module to the Tab
             dataProvider.MoveTabModule(fromTabId, moduleId, toTabId, toPaneName, UserController.Instance.GetCurrentUserInfo().UserID);
 
+            //Update the Tab reference for the module's ContentItems
+            var contentController = Util.GetContentController();
+            var contentItems = contentController.GetContentItemsByModuleId(moduleId);
+            if (contentItems?.Count() > 0)
+            {
+                foreach (var item in contentItems)
+                {
+                    if (item.TabID != toTabId)
+                    {
+                        item.TabID = toTabId;
+                        contentController.UpdateContentItem(item);
+                    }
+                }
+            }

```

Comment:
Do we need to check the count? If we are just going into a foreach that foreach will just get skipped if there are no items right?